packages/native/src/TxNative.js

```diff
@@ -153,6 +154,7 @@ export default class TxNative {
         response = await axios.get(`${this.cdsHost}/content/${localeCode}`, {
           headers: {
             Authorization: `Bearer ${this.token}`,
+            'X-NATIVE-SDK': `txjs/${VERSION}`,
```

Comment:
How is this variable get assigned its value? Why not use `package.json` here too?

Reply:
Because this is web packbuild, I do not want to require and import the entire package.json into the build and end-up in a browser :P

The build-in `webpack.DefinePlugin` exists for this kind of staff. You define some global constants and make them available in the source code. See `webpack.common.js` on how this is assigned.